examples/MQ/pixelDetector/run/scripts/test-pixelSim.sh.in

```diff
@@ -51,37 +51,44 @@ dds-topology -s $DDS_SESSION_ID --activate $FAIRMQ_DDS_TOPO_FILE --disable-valid
 
 sleep 1
 
-fairmq-dds-command-ui -w "IDLE"
-fairmq-dds-command-ui -c i
-fairmq-dds-command-ui -c k
-fairmq-dds-command-ui -c b
-fairmq-dds-command-ui -c x
-fairmq-dds-command-ui -c j
-fairmq-dds-command-ui -c r
+for cmd in IDLE i k b x j r
+do
+	echo ""
+	if (( ${#cmd} == 1 ))
+	then
+		echo "***** fairmq-dds-command-ui -c $cmd"
+		fairmq-dds-command-ui -c "$cmd"
+	else
+		echo "***** Waiting for state $cmd"
+		time fairmq-dds-command-ui -w "$cmd"
+	fi
+done
```

Comment:
In FairMQ < `1.4.13` this needs to be

```suggestion
fairmq-dds-command-ui -w "IDLE" -n ${requiredNofAgents}
fairmq-dds-command-ui -c i -w "INITIALIZING DEVICE" -n ${requiredNofAgents}
fairmq-dds-command-ui -c k -w "INITIALIZED" -n ${requiredNofAgents}
fairmq-dds-command-ui -c b -w "BOUND" -n ${requiredNofAgents}
fairmq-dds-command-ui -c x -w "DEVICE READY" -n ${requiredNofAgents}
fairmq-dds-command-ui -c j -w "READY" -n ${requiredNofAgents}
fairmq-dds-command-ui -c r
```

The `-n` argument expects the number of device addressed.

Reply:
See https://github.com/FairRootGroup/FairMQ/blob/93eb599df7d9ee4c869e2fe70c03b7a845739683/examples/dds/fairmq-start-ex-dds.sh.in#L57-L72